esp32-example.yaml

```
@@ -224,15 +240,15 @@ sensor:
   #  11  Status/Fault flag                     2 byte   R  uint16  Hex See ^3
   - platform: modbus_controller
     modbus_controller_id: bms0
-    name: "${name} errors bitmask"
+    name: "${name} fault status bitmask"
```

```suggestion
    name: "${name} errors bitmask"
```

I try to avoid a breaking change here.